packages/commerce-sdk-react/src/hooks/ShopperBaskets/mutation.ts

```diff
@@ -304,10 +330,7 @@ export const getCacheUpdateMatrix = (customerId: string | null) => {
             params: Argument<Client['mergeBasket']>,
             response: DataType<Client['mergeBasket']>
```

Comment:
Do we not get lint errors for unused method arguments in TS?

Reply:
We currently do not set TS to prompt error for it.